dcos/package.py

```diff
@@ -260,6 +263,17 @@ def is_match(app):
             logger.info(
                 'Found the following frameworks: {}'.format(framework_ids))
 
+            # Emit post uninstall notes
+            emitter.publish(
+                DefaultError(
+                    'Uninstalled package [{}] version [{}]'.format(
+                        package_json.get('name'),
+                        package_json.get('version'))))
```

Comment:
We should probably put the `name`/`version` in an `if` or else the message wouldn't make sense